App/Sagas/StartupSagas.js

```diff
@@ -35,6 +35,6 @@ export function * startup () {
 
   // Dispatch actions you want on STARTUP
   // yield put(IpfsNodeActions.createNodeRequest(RNFS.DocumentDirectoryPath))
-  const previousState = yield select(IpfsNodeSelectors.appState)
-  yield put(IpfsNodeActions.appStateChange(previousState, AppState.currentState))
```

Comment:
Triggering this from `TextileManager` now.

Reply:
Is it still safe to have this here @asutula? Sticking this back in fixes the error on android.